checkstyle/import-control.xml

```diff
@@ -37,5 +38,7 @@
     <allow pkg="feign"/>
     <allow pkg="io.grpc" />
     <allow pkg="info.ganglia" />
+    <allow pkg="org.gaul.s3proxy" />
```

Comment:
What I dislike about this is that it is opening up for all classes to import it, even though we only declare as a test compile dependency. To narrow down the scope of the class in this package with respect to checkstyle, we would need to create a separate import control file only for test classes. Please create an issue for us to do it.